tests/test_module_lesson_attendance.py

```diff
@@ -0,0 +1,11 @@
+def test_student_self_attendance_registration(client, auth):
```

Comment:
Your test cases are not complete.
you didn't test instances where the actor successfully checkin etc.
this is where you exercise the entirety of your code use case.

just for the sake of this project.
just add an additional test where the student successfully checkin.

Reply:
@nahloo-obi please consider adding an additional test case where a student is able to successfully check in